code/ui/components/src/components/typography/DocumentWrapper.stories.tsx

```diff
@@ -1,19 +1,12 @@
 import React from 'react';
 
 import { DocumentWrapper } from './DocumentWrapper';
-import MarkdownSample from './DocumentFormattingSample.mdx';
 
 export default {
   component: DocumentWrapper,
   decorators: [(storyFn: any) => <div style={{ width: '600px' }}>{storyFn()}</div>],
 };
 
-export const WithMarkdown = () => (
-  <DocumentWrapper>
-    <MarkdownSample />
-  </DocumentWrapper>
-);
```

Comment:
AFAICS this is completely redundant. The story below this tests the exact same thing.

I'm removing this because it adds no value, and can't work without the mdx-plugin.
The MDX-plugin (for vite) is now added by addon-docs, which is obviously removed when specifying the `--test` flag,